pkg/controller/machine.go

```diff
@@ -391,7 +392,7 @@ func (c *controller) machineCreate(machine *v1alpha1.Machine, driver driver.Driv
 		clone.Spec.ProviderID = actualProviderID
 		machine, err = c.controlMachineClient.Machines(clone.Namespace).Update(clone)
 		if err != nil {
-			glog.Warningf("Machine update failed. Retrying, error: %s", err)
+			glog.Warningf("Machine UPDATE failed. Retrying, error: %s", err)
```

Comment:
Could the name of the machine be useful in the logs?

Reply:
Yes, makes sense. Made suggested changes. Refer - https://github.com/gardener/machine-controller-manager/pull/330/commits/c3b4dec3a8cd10096d8464fc3550999a630571d9.